src/app/util/cluster-config.h

```diff
@@ -0,0 +1,47 @@
+/**
+ *    Copyright (c) 2025 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+#include <lib/core/DataModelTypes.h>
+#include <lib/support/BitFlags.h>
+
+#include <cstdint>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace EmberApplicationConfig {
+
+/// An enumeration acting as a placeholder of "no features defined
+/// for a cluster, enforcing that the feature map type is always
+/// a form of bitflags"
+enum NoFeatureFlagsDefined : uint32_t
```

Comment:
Why aren't we using NoFeatureFlagsDefined in the generated bits, then?

Reply:
This seems like a miss will update.

Reply:
Updated.